WalletWasabi.Fluent/Icons/Icons.axaml

```diff
@@ -503,6 +503,7 @@
       <StreamGeometry x:Key="pause_regular">M17.75,7 C19.5449254,7 21,8.45507456 21,10.25 L21,37.75 C21,39.5449254 19.5449254,41 17.75,41 L12.25,41 C10.4550746,41 9,39.5449254 9,37.75 L9,10.25 C9,8.45507456 10.4550746,7 12.25,7 L17.75,7 Z M35.75,7 C37.5449254,7 39,8.45507456 39,10.25 L39,37.75 C39,39.5449254 37.5449254,41 35.75,41 L30.25,41 C28.4550746,41 27,39.5449254 27,37.75 L27,10.25 C27,8.45507456 28.4550746,7 30.25,7 L35.75,7 Z M17.75,9.5 L12.25,9.5 C11.8357864,9.5 11.5,9.83578644 11.5,10.25 L11.5,37.75 C11.5,38.1642136 11.8357864,38.5 12.25,38.5 L17.75,38.5 C18.1642136,38.5 18.5,38.1642136 18.5,37.75 L18.5,10.25 C18.5,9.83578644 18.1642136,9.5 17.75,9.5 Z M35.75,9.5 L30.25,9.5 C29.8357864,9.5 29.5,9.83578644 29.5,10.25 L29.5,37.75 C29.5,38.1642136 29.8357864,38.5 30.25,38.5 L35.75,38.5 C36.1642136,38.5 36.5,38.1642136 36.5,37.75 L36.5,10.25 C36.5,9.83578644 36.1642136,9.5 35.75,9.5 Z</StreamGeometry>
       <StreamGeometry x:Key="stop_regular">M19.25 4.5C19.3881 4.5 19.5 4.61193 19.5 4.75V19.25C19.5 19.3881 19.3881 19.5 19.25 19.5H4.75C4.61193 19.5 4.5 19.3881 4.5 19.25V4.75C4.5 4.61193 4.61193 4.5 4.75 4.5H19.25ZM4.75 3C3.7835 3 3 3.7835 3 4.75V19.25C3 20.2165 3.7835 21 4.75 21H19.25C20.2165 21 21 20.2165 21 19.25V4.75C21 3.7835 20.2165 3 19.25 3H4.75Z</StreamGeometry>
 
+      <StreamGeometry x:Key="undo_icon_regular">M492.373,58.793c-5.76-0.96-11.307,2.987-12.267,8.747l-11.2,66.88C425.707,55.38,341.867,5.353,249.28,5.353 C111.787,5.353,0,115.433,0,250.687S111.893,496.02,249.387,496.02c93.227,0,178.027-50.56,221.227-131.84 c2.773-5.227,0.747-11.627-4.48-14.4c-5.227-2.773-11.627-0.747-14.4,4.48c-39.467,74.347-117.013,120.533-202.347,120.533 c-125.76-0.107-228.053-100.587-228.053-224.107s102.293-224,227.947-224c86.933,0,165.333,48.107,203.947,123.733l-77.867-14.933 c-5.76-1.067-11.307,2.773-12.373,8.64c-1.067,5.653,2.667,11.2,8.32,12.373l101.227,19.413c0.64,0.107,1.28,0.213,2.027,0.213 c5.227,0,9.6-3.733,10.56-8.853l16.107-96C502.187,65.407,498.347,59.86,492.373,58.793z</StreamGeometry>
```

Comment:
We use icons from the fluent icon pack https://github.com/microsoft/fluentui-system-icons/blob/master/icons_regular.md

Although I don't think fluent one is a usual undo icon. What do you think?

Reply:
@soosr I like the current icon much more:

![image](https://user-images.githubusercontent.com/98904713/159191739-15a4b2c7-ef75-46fd-a868-e05743a79f29.png)

Versus the fluentui one:

![image](https://user-images.githubusercontent.com/98904713/159191797-36749b8d-bd2a-4e7a-93b7-404a41414524.png)

let's keep the current one if you don't mind ;)

Reply:
Just my average user opinion, but this 👇

![Image](https://user-images.githubusercontent.com/98904713/159191739-15a4b2c7-ef75-46fd-a868-e05743a79f29.png) 

looks like a reload icon instead of uno :)

Reply:
I agree with @SuperJMN it looks more like a refresh icon. The fluent one looks more like undo icon.

Reply:
@ichthus1604 Could you fix the icon? I would merge this PR

Reply:
@soosr done